tests/drivers/fuel_gauge/bq27z746/src/test_bq27z746.c

```diff
@@ -42,10 +42,7 @@ ZTEST_USER_F(bq27z746, test_get_all_props_failed_returns_negative)
 
 	int ret = fuel_gauge_get_props(fixture->dev, props, ARRAY_SIZE(props));
 
-	zassert_equal(props[0].status, -ENOTSUP, "Getting bad property %d has a good status.",
-		      props[0].property_type);
-
-	zassert_true(ret < 0);
+	zassert_equal(ret, -ENOTSUP);
```

Comment:
Note - I think `-EINVAL` is probably a more appropriate error code, but I see that the existing drivers use `-ENOTSUP`.

Reply:
I think it's not entirely clear which one to use here. As you mentioned, sensor version uses `-ENOTSUP` and there may be properties that are entirely valid but just aren't supported by this driver.